LayoutTest/TestCase/LYTLayoutFailingTestSnapshotRecorder.m

```diff
@@ -0,0 +1,228 @@
+// © 2016 LinkedIn Corp. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+#import "LYTLayoutFailingTestSnapshotRecorder.h"
+#import "LYTConfig.h"
+#import "LYTLayoutTestCase.h"
+
+void SimpleLog(NSString *format, ...) {
+    // Log to the console without timestamp that NSLog gives us
```

Comment:
The timestamp made the logging look like part of the output of the tests, whereas this is a summary of the tests run, similar to "Test session log:". The timestamp isn't needed and obscures the essential information.